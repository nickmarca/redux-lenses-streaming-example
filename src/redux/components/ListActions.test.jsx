import React from 'react';
import { create } from 'react-test-renderer';
import ListActions from './ListActions';

describe('ListActions', () => {
    it('matches the snapshot', () => {
        const listActions = create(<ListActions />);

        expect(listActions.toJSON()).toMatchSnapshot();
    });
});
