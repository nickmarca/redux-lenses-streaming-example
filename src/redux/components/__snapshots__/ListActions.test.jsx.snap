// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListActions matches the snapshot 1`] = `
<div
  className="panel-block"
>
  <div
    className="control"
  >
    <button
      className="button is-small is-info"
      style={
        Object {
          "marginRight": "10px",
        }
      }
    >
      Subscribe
    </button>
    <button
      className="button is-small is-danger"
    >
      Clear Messages
    </button>
  </div>
  <div
    className="control"
  />
</div>
`;
